dependencies.gradle

```diff
@@ -0,0 +1,63 @@
+ext {
```

Comment:
Czy jest jakiś sposób żeby nie tracić tego co oferuję nam IDE, które podpowiada że są jakieś biblioteki które można podbić do wyższej wersji ? AndroidStudio chyba nie radzi sobie z tym jeżeli nie mamy tego podefiniowanego w build.gradle.

Reply:
Niestety nie znam takiego rozwiązania. Sposób na pewno jakiś jest. To co mi przychodzi do głowy to pobawienie się przy konfiguracji linta, ale osobiście tego nie robiłem.

Reply:
@keresmi
Ostatnio na szkoleniu prowadzący też zwrócił uwage że albo jest opcja wydzielania stałych w build.gradle i implementowaniu zależności - wtedy IDE sobie radzi albo robienie takiej mapy w ext jak zaproponowałeś. Osobiście uważam twoje rozwiązanie za bardziej schludne, a podbiciem biblioteki i tak podejmuje się ŚWIADOMIE developer.